pkg/deployment/resources/pod_cleanup.go

```diff
@@ -49,21 +46,32 @@ func (r *Resources) CleanupTerminatedPods(ctx context.Context, cachedStatus insp
 
 	// Update member status from all pods found
 	status, _ := r.context.GetStatus()
+
 	err := cachedStatus.IteratePods(func(pod *v1.Pod) error {
 		if k8sutil.IsArangoDBImageIDAndVersionPod(pod) {
 			// Image ID pods are not relevant to inspect here
 			return nil
 		}
 
-		if !(k8sutil.IsPodSucceeded(pod) || k8sutil.IsPodFailed(pod) || k8sutil.IsPodTerminating(pod)) {
-			return nil
-		}
+		//if !(k8sutil.IsPodSucceeded(pod, coreContainers) || k8sutil.IsPodFailed(pod, coreContainers) ||
```

Comment:
what about here, can I do it? 
is it possible that member status is unknown and pod can be succeeded or failed or terminating?